src/constants/tokens/vBep/testnet.ts

```diff
@@ -228,4 +228,191 @@ export const TESTNET_VBEP_TOKENS = {
     asset: vXvs, // TODO: add correct asset
     underlyingToken: TESTNET_TOKENS.usdd,
   } as VToken,
+  // Pool 1
+  '0x9b921bbcdc18030540bcb341b1fec7fa00f7bee5': {
+    address: '0x9b921bbcdc18030540bcb341b1fec7fa00f7bee5',
+    decimals: 8,
+    symbol: 'vUSDD',
+    asset: vXvs, // TODO: add correct asset
+    underlyingToken: TESTNET_TOKENS.usdd,
+  } as VToken,
+  '0x225a34c5f03b31f9f4648d9c4eee5191c87d1b25': {
+    address: '0x225a34c5f03b31f9f4648d9c4eee5191c87d1b25',
+    decimals: 8,
+    symbol: 'vUSDT',
+    asset: vXvs, // TODO: add correct asset
+    underlyingToken: TESTNET_TOKENS.usdt,
+  } as VToken,
+  '0x91f90e30b1695218d1ecbdc41177133d9de2b094': {
+    address: '0x91f90e30b1695218d1ecbdc41177133d9de2b094',
+    decimals: 8,
+    symbol: 'vWOO',
+    asset: vXvs, // TODO: add correct asset
+    underlyingToken: TESTNET_TOKENS.woo,
+  } as VToken,
+  '0x1723958488d5acd4f2db5e5a4efd7c4e07973425': {
+    address: '0x1723958488d5acd4f2db5e5a4efd7c4e07973425',
+    decimals: 8,
+    symbol: 'vALPACA',
+    asset: vXvs, // TODO: add correct asset
+    underlyingToken: TESTNET_TOKENS.alpaca,
+  } as VToken,
+  '0x7250b36b8971adf911f0d82c162634de684fc9b3': {
+    address: '0x7250b36b8971adf911f0d82c162634de684fc9b3',
+    decimals: 8,
+    symbol: 'vBSW',
+    asset: vXvs, // TODO: add correct asset
+    underlyingToken: TESTNET_TOKENS.bsw,
+  } as VToken,
+  '0xa7a4c0f7fabd808a4336c71f15d27a2efced5c70': {
+    address: '0xa7a4c0f7fabd808a4336c71f15d27a2efced5c70',
+    decimals: 8,
+    symbol: 'vBIFI',
+    asset: vXvs, // TODO: add correct asset
+    underlyingToken: TESTNET_TOKENS.bifi,
+  } as VToken,
+  // Pool 2
+  '0x6d258734b003e803c1f67ccaa2ccddab93a60067': {
+    address: '0x6d258734b003e803c1f67ccaa2ccddab93a60067',
+    decimals: 8,
+    symbol: 'vUSDD',
+    asset: vXvs, // TODO: add correct asset
```

Comment:
Do we still need vToken logos?

Reply:
We don't indeed, since they aren't used anywhere. I've created a ticket to get them removed: https://jira.toolsfdg.net/browse/VEN-1511